src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/ErrorMsg.java

```diff
@@ -48,7 +45,7 @@ public String getMsgId() {
 
     @Override
     public boolean isTestOnly() {
-        return this.testOnly;
+        return false;
```

Comment:
Do we need this anymore ?

Reply:
Yeah, it's in the interface that we implement